Add wiki pages for identity v1.1 (#1414)
iotaledger/iota-wiki: docs/build/identity.rs/1.1/docs/getting-started/rust.mdx

---
sidebar_label: Rust
description: Getting started with the IOTA Identity Rust Library.
image: /img/Identity_icon.png
keywords:
  - Rust
  - Identity
---
# Getting Started with Rust

## Requirements

- [Rust](https://www.rust-lang.org/) (>= 1.62)
- [Cargo](https://doc.rust-lang.org/cargo/) (>= 1.62)

## Include the Library

> To include IOTA Identity in your project add it as a dependency in your `Cargo.toml`:

Reviewer: ```suggestion
To include IOTA Identity in your project, add it as a dependency in your `Cargo.toml`:
```

### Latest Stable Release

This version is published to crates.io and is **stable**, following semantic versioning.

```toml
[dependencies]

> identity_iota = { version = "1.0.0" }

Reviewer: ```suggestion
identity_iota = { version = "1.1.0" }
```

```

### Development Release

> This version matches the `main` branch of this repository. It has all the **latest features**, but as such it **may also have undocumented breaking changes**.

Reviewer: ```suggestion
This version matches the `main` branch of this repository. It has all the **latest features**, but as such, it **may also have undocumented breaking changes**.
```

```toml
[dependencies]
identity_iota = { git = "https://github.com/iotaledger/identity.rs", branch = "main"}
```

## Examples

To try out the [examples](https://github.com/iotaledger/identity.rs/tree/v1.1.0/examples), you should:

1. Clone the repository:

```bash
git clone https://github.com/iotaledger/identity.rs
```

2. Build the repository:

```bash
cargo build
```

3. Run your first example:

```bash
cargo run --example getting_started
```

## API Reference

> The API reference for the Rust library can be found on [docs.rs](https://docs.rs/identity_iota/latest/identity_iota/index.html).

Reviewer: ```suggestion
You can find the API reference for the Rust library on [docs.rs](https://docs.rs/identity_iota/latest/identity_iota/index.html).
```

> If you would like to build the documentation locally you can do so with the following command:

Reviewer: ```suggestion
If you would like to build the documentation, locally you can do so with the following command:
```
